src/components/templates/Audience/Audience.tsx

```diff
@@ -421,9 +419,7 @@ export const Audience: React.FunctionComponent = () => {
                                     isOnRight ? "left" : "right"
                                   }
                                   avatarClassName={"profile-avatar"}
-                                  setSelectedUserProfile={
-                                    setSelectedUserProfile
-                                  }
+                                  setSelectedUserProfile={setUserProfile}
```

Comment:
I expect that this prop can probably be removed entirely, and just use the hook directly within the component.